torchrl/envs/utils.py

```diff
@@ -155,6 +155,13 @@ def step_mdp(
             is_shared=False)
 
     """
+    if isinstance(done_key, tuple) and len(done_key) == 1:
+        done_key = done_key[0]
+    if isinstance(reward_key, tuple) and len(reward_key) == 1:
+        reward_key = reward_key[0]
+    if isinstance(action_key, tuple) and len(action_key) == 1:
+        action_key = action_key[0]
```

Comment:
this is expensive let's not do that if we can avoid it
We could blend this in unravel_keys in tensordict (which is coded in c++) if it's there for efficiency purposes